shared/src/commonMain/kotlin/data/credentials/EuPidCredentialAdapter.kt

```diff
@@ -219,20 +221,24 @@ sealed class EuPidCredentialAdapter(
             storeEntry: SubjectCredentialStore.StoreEntry,
             decodePortrait: (ByteArray) -> ImageBitmap?,
         ): EuPidCredentialAdapter {
-            if (storeEntry.scheme !is EuPidScheme) {
-                throw IllegalArgumentException("credential")
+            if (storeEntry.scheme !is EuPidScheme && storeEntry.scheme !is EuPidSdJwtScheme) {
+                throw IllegalArgumentException("credential: ${storeEntry.scheme}")
             }
             return when (storeEntry) {
                 is SubjectCredentialStore.StoreEntry.Vc ->
                     (storeEntry.vc.vc.credentialSubject as? EuPidCredential)
-                        ?.let { EuPidCredentialVcAdapter(it, decodePortrait) }
+                        ?.let { EuPidCredentialVcAdapter(it, decodePortrait, storeEntry.scheme!!) }
```

Comment:
won't this potentially break during an update of the app? (and the next change too)

Reply:
No, it won't, since `EuPidcredentialVcAdapter` is a class internal to Valera, and also the `storeEntry.scheme` property has been there all along.